eq_translations/extractable_strings.py

```diff
@@ -59,6 +63,10 @@
         "json_path": "$..summary.add_link_text",
         "description": "List collector add link text",
     },
+    {
+        "json_path": "$..summary.item_label",
+        "description": "List collector item label",
+    },
```

Comment:
Are these and the ones above used?. Looking at validation, only `titel` and `item_title` is needed. Those would be removed in future too.

Reply:
Leave it for now. We'll look to remove, when we remove `block.summary`. It doesn't hurt for now.